packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -1217,4 +1217,168 @@ describe('GCFBootstrapper', () => {
       sinon.assert.calledOnce(configStub);
     });
   });
+
+  describe('enqueueTask', () => {
+    it('queues a GCF URL', async () => {
+      const bootstrapper = new GCFBootstrapper({
+        projectId: 'my-project',
+        functionName: 'my-function',
+        location: 'my-location',
+      });
+      const createTask = sandbox
+        .stub(bootstrapper.cloudTasksClient, 'createTask')
+        .resolves();
+      await bootstrapper.enqueueTask({
+        body: JSON.stringify({installation: {id: 1}}),
+        id: 'some-request-id',
+        name: 'event.name',
+      });
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore - https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36436
```

Comment:
Instead of using `@ts-ignore` can we use a cast to `any`?  `@ts-ignore` comes with a bunch of other side effects I think we want to avoid in this case.

Reply:
Here and below